plugins/kubernetes/app/models/kubernetes/role_validator.rb

```diff
@@ -103,6 +104,17 @@ def self.keep_name?(e)
 
     private
 
+    def validate_load_balancer
+      allowed = ENV["KUBERNETES_ALLOWED_LOAD_BALANCER_NAMESPACES"].to_s.split(",")
+      return if allowed.empty?
+      bad = @elements.map do |e|
+        next unless e[:kind] == "LoadBalancer"
```

Comment:
@grosser What is the APIGroup of this kind?
Usually LoadBalancers are part of the service (i.e. kind = 'Service' and spec.type = 'LoadBalancer') 
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#servicespec-v1-core

Reply:
yeah ... thx for paying attention :D